pkg/chartutil/create.go

```diff
@@ -159,28 +160,30 @@ metadata:
 {{ include "<CHARTNAME>.labels" . | indent 4 }}
   {{- with .Values.ingress.annotations }}
   annotations:
-{{ toYaml . | indent 4 }}
-{{- end }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
```

Comment:
Decrease by 2 indents so `{{- end }}` is inline with `{{- with .Values.ingress.annotations }}`

Reply:
Ok，i will do it now，i am sorry for that。